src/locales/zh.json

```diff
@@ -516,6 +516,7 @@
       "script_data": "data",
       "export-transaction": "导出交易",
       "capacity_usage": "Capacity 使用情况",
+      "utxo_graph": "UTXO 可视化",
```

Comment:
IMO, `UTXO 可视化` doesn't match `UXTO graph` because `graph` has a specific meaning of `graph theory` while `可视化` is for general visualization.

I will expect `UTXO 可视化` to be `visualization of a specific UTXO/cell` instead of the relationship between `UTXO`

Any suggestion from @Sven-TBD